src/calendar/header/index.tsx

```diff
@@ -206,19 +209,26 @@ const CalendarHeader = forwardRef((props: CalendarHeaderProps, ref) => {
     const renderArrowDirection = isLeft ? 'left' : 'right';
     const shouldDisable = isLeft ? disableArrowLeft : disableArrowRight;
 
+    const hitSlop: Insets | undefined =
```

Comment:
Please use memo here to avoid creating this on every render

Reply:
Added useMemo